lore/end_users/use/README.md

```diff
@@ -34,3 +35,41 @@ Now we have a pull request that has core contributor approval and successful che
 This image illustrates the BonnyCI continuous integration pipeline.
 
 ![BonnyCI Workflow](../../misc/images/BonnyCIWorkflow.png)
+
+## Handling Merge Failures
+
+Anne-Bonny, the bot user which handles checking and gating pull requests, may
+report back to a pull request informing the developer and reviewers that she
+encountered a merge failure. This typically happens in the gate queue, and will
+usually fall into one of the two categories detailed below.
+
+### Stale Pull Requests
+
+Occasionally, a pull request may fall by the wayside while other changes are
+given a higher priority by reviewers. This can leave behind a pull request
+which has become stale, meaning that the changes it is making are based off of
+an older git history. In most cases, this can be resolved with a simple rebase:
+
+```shell
+  $ git fetch <remote>
+  $ git rebase <remote>/master
+  $ git push origin/<branch>
+```
+
+This will update the pull request's branch by taking its commit(s) off of the
+branch's HEAD, updating the branch to the new compare branch's HEAD, and then
+placing the pull request's commit(s) back on top. This usually results in a
+branch that is n number of commits ahead of origin/master, where n is the
+number of commits in your pull request (preferably one).
+
+### Competing Pull Requests
```

Comment:
I was trying to differentiate between github saying "this PR's branch is out of date" with "this file has now changed and I'm not sure what to do with it". I changed up the wording in the second section to try to reflect that a little more.